pkg/apis/v1alpha1/networkinfo_types.go

```diff
@@ -0,0 +1,56 @@
+/* Copyright © 2024 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+//+kubebuilder:object:root=true
+//+kubebuilder:storageversion
+
+// NetworkInfo defines network infos for a namespace.
+type NetworkInfo struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   NetworkInfoSpec   `json:"spec,omitempty"`
+	Status NetworkInfoStatus `json:"status,omitempty"`
+}
+
+//+kubebuilder:object:root=true
+
+// NetworkInfoList contains a list of NetworkInfo.
+type NetworkInfoList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []NetworkInfo `json:"items"`
+}
+
+// NetworkInfoSpec defines VPC configuration.
```

Comment:
We need not a spec and so it can be removed now?

Reply:
sure.